### PR TITLE
fix: @babel/polyfill not found in Docker container

### LINKED ISSUE #829: Latest Docker image throws: Error: Cannot find module '@babel/polyfill'
#### Description

Describe the issue or feature request in detail. Take a screenshot or record a video will be helpful.

#### Versions

- CNCjs: 1.10.2 (Docker latest)
- Node.js: 6.x
- NPM: 5.x

#### How Do You Install CNCjs?

- [ ] NPM
- [ ] Download the CNCjs Desktop Application
- [x] Docker

#### CNC Controller

- [x] Grbl
- [ ] Smoothieware
- [ ] TinyG/g2core

#### Hardware

- [ ] Raspberry Pi
- [x] Desktop or Laptop
- [ ] Mobile Device

#### Operating System

- [ ] Not Applicable
- [x] Windows
- [ ] Mac
- [x] Linux

I am using the cncjs:latest docker container. 
(Also present in older images)
The container throws following error on start:

`internal/modules/cjs/loader.js:905
throw err;
^

Error: Cannot find module '@babel/polyfill'
Require stack:

    /opt/cncjs/entrypoint
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
    at Function.Module._load (internal/modules/cjs/loader.js:746:27)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:101:18)
    at Object. (/opt/cncjs/entrypoint:3:1)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:75:12) {
    code: 'MODULE_NOT_FOUND',
    requireStack: [ '/opt/cncjs/entrypoint' ]
    }`

Does the Dockerimage use an old CNCJS version?

### PR DESCRIPTION
**Pull request by emcniece**
Applies the fix from https://github.com/cncjs/cncjs/pull/803 to the Docker entrypoint script so that the Docker images can run properly.

This is not fully functional yet: when accessing the app from the browser an error displays and the container logs out:

```
2023-05-17T03:29:37.666Z - info init Loading configuration from "/root/.cncrc"
(node:1) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
2023-05-17T03:29:37.725Z - info init Starting the server at http://172.17.0.2:8000
Error: Failed to lookup view "index.hbs" in views directories "/opt/cncjs/app" or "/opt/cncjs/server/views"
    at Function.render (/opt/cncjs/node_modules/express/lib/application.js:580:17)
    at ServerResponse.render (/opt/cncjs/node_modules/express/lib/response.js:1008:7)
    at /opt/cncjs/server/app.js:66:9
    at Layer.handle [as handle_request] (/opt/cncjs/node_modules/express/lib/router/layer.js:95:5)
    at next (/opt/cncjs/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/opt/cncjs/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/opt/cncjs/node_modules/express/lib/router/layer.js:95:5)
    at /opt/cncjs/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/opt/cncjs/node_modules/express/lib/router/index.js:335:12)
    at next (/opt/cncjs/node_modules/express/lib/router/index.js:275:10)
    at Layer.handle [as handle_request] (/opt/cncjs/node_modules/express/lib/router/layer.js:91:12)
    at trim_prefix (/opt/cncjs/node_modules/express/lib/router/index.js:317:13)
    at /opt/cncjs/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/opt/cncjs/node_modules/express/lib/router/index.js:335:12)
```

To remedy this, `index.hbs` is copied into the server directory. Now when accessing the app in browser a loading icon appears, but nothing happens. There are no terminal or JS console logs.

I'm not sure if this is the correct approach for fixing the HTML templating, need to investigate. @cheton any suggestions? 🙏 

Resolves #829 and #826.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -28,6 +28,7 @@ RUN git clone https://github.com/nvm-sh/nvm.git "$NVM_DIR" \
 
 COPY ./dist/cncjs $BUILD_DIR/cncjs
 COPY ./entrypoint $BUILD_DIR/cncjs/
+COPY ./index.hbs $BUILD_DIR/cncjs/server/views
```
Comment (cheton) [inline Dockerfile:31]: You don't need to copy the file. The file will be automatically copied to the `server/views` directory within the the [build-prod.sh](https://github.com/cncjs/cncjs/blob/master/scripts/build-prod.sh#L29) script.

File: entrypoint [hunk 2]
```diff
@@ -1,8 +1,6 @@
 #!/usr/bin/env node
 
-require('@babel/polyfill');
-
-const launchServer = require('/opt/cncjs/server-cli').default;
+const launchServer = require('/opt/cncjs/server-cli');
```
Comment (cheton) [inline entrypoint:3]: Looks good to me

File: Dockerfile (hunk 1 above)
Comment (cheton) [reply Dockerfile:31]: The `index.hbs` file, which serves as the HTML template, will be copied from the root directory to the output path `dist/cncjs/app` as defined in `webpack.config.production.js`:

```js
    new HtmlWebpackPlugin({
      filename: 'index.hbs',
      template: path.resolve(__dirname, 'index.hbs'),
      chunksSortMode: 'dependency' // Sort chunks by dependency
    })
```

image

Upon further reflection, I have come to understand that the build process for building a Docker image is not straightforward and is divided into several build scripts. Nevertheless, `index.hbs` has been removed and updated in the `next` branch.

Comment (cheton) [reply Dockerfile:31]: TL;DR:

I think you can leave the Dockerfile unchanged in this PR. Once the PR merged, I will trigger a CircleCI build to update existing Docker image with the tag 1.10.2 to fix this issue.

**Review by cheton — APPROVED** [review]

**Comment by emcniece** [comment]
Thank you for the help!

**Pull request merged**